scripts/userstyles.yml

```
@@ -761,6 +763,13 @@ userstyles:
     readme:
       app-link: "https://app.raindrop.io"
     current-maintainers: [*thismoon]
+  readwise:
+    name: Readwise
+    categories: [note_taking, education]
+    color: mauve
```

```suggestion
    color: blue
```
Either `blue` or `text` would make more sense here - the color is supposed to match the theme/primary color of the website.